src/components/MenuItem.js

```diff
@@ -113,7 +118,7 @@ const MenuItem = ({
                 </View>
                 {shouldShowRightIcon && (
                 <View style={styles.createMenuIcon}>
-                    <Icon src={iconRight} fill={getIconFillColor(getButtonState(hovered, pressed))} />
+                    <Icon src={iconRight} fill={getIconFillColor(getButtonState(focused || hovered, pressed))} />
```

Comment:
Should we also add this here -> 

https://github.com/Expensify/Expensify.cash/blob/c1536a83ef628f27dcae5d0320bfba16d7bff58e/src/components/MenuItem.js#L81

Reply:
also NAB, but I think this should be updated to something like `isActive` + `CONST.BUTTON_STATES.ACTIVE` instead of `isHovered` since `isHovered` can now mean "hovered or focused" which doesn't seem like the original intention.

https://github.com/Expensify/Expensify.cash/blob/c1536a83ef628f27dcae5d0320bfba16d7bff58e/src/libs/getButtonState.js#L11-L22